django/contrib/gis/db/backends/mysql/operations.py

```diff
@@ -29,12 +31,8 @@ def is_mysql_5_6(self):
         return self.connection.mysql_version < (5, 7, 6)
 
     @cached_property
-    def uses_invalid_empty_geometry_collection(self):
```

Comment:
I think you're right, I decided to move it to `features` and rename it as its meaning changed a bit.